pkg/config/application.go

```diff
@@ -26,20 +26,13 @@ const (
 	defaultWaitApprovalTimeout  = Duration(6 * time.Hour)
 	defaultAnalysisQueryTimeout = Duration(30 * time.Second)
 	allEventsSymbol             = "*"
-
-	EnvLabelKey = "env"
 )
 
 type GenericApplicationSpec struct {
 	// The application name.
 	// This is required if you set the application through the application configuration file.
 	Name string `json:"name"`
-	// The environment name. You need to make sure that the environment name is unique in your project.
-	// Deprecated.
-	EnvName string `json:"envName"`
```

Comment:
How about just keeping this field to ensure that all kinds of config could be loadable?

Reply:
Because currently unknown fields are being disallowed while loading the config.

Reply:
I would say `json.Unmarshal()` is able to parse one that has unknown field like `envName`, but am I correct?

Reply:
https://go.dev/play/p/11XRE19yJv2

Reply:
I thought as same as @nakabonne even without that field, json.Unmarshal() could work, right?

Reply:
https://github.com/pipe-cd/pipe/blob/master/pkg/config/config.go#L174

Reply:
We are having a customized one.

Reply:
looks like it works 👍 
https://go.dev/play/p/X5pA17Zo0lQ

Reply:
Oops!